package.json

```diff
@@ -10,8 +10,8 @@
     "url": "https://github.com/OpenHumans/open-humans.git"
   },
   "engines": {
-    "node": "6.x",
-    "npm": "3.x"
+    "node": "6.x || 7.x || 8.x || 9.x",
```

Comment:
Can we keep the versions as is? I guess at least we don't need these bumps for the project grants page and by keeping this constant we have a better atomic commit.